packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -163,4 +163,32 @@ class AndroidWorkflow extends DoctorValidator implements Workflow {
     // Success.
     return new ValidationResult(ValidationType.installed, messages, statusInfo: sdkVersionText);
   }
+
+  /// Run the Android SDK manager tool in order to accept SDK licenses.
+  static Future<bool> runLicenseManager() async {
+    if (androidSdk == null) {
+      printStatus('Unable to locate Android SDK.');
+      return false;
+    }
+
+    // If we can locate Java, then add it to the path used to run the Android SDK manager.
+    final Map<String, String> sdkManagerEnv = <String, String>{};
+    final String javaBinary = _findJavaBinary();
+    if (javaBinary != null) {
+      sdkManagerEnv['PATH'] =
+          platform.environment['PATH'] + os.pathVarSeparator + fs.path.dirname(javaBinary);
+    }
+
+    final Process process = await Process.start(
```

Comment:
This should be using processManager, or more likely, methods in `base/process.dart`